umm-spec-lib/src/cmr/umm_spec/test/umm_generators.clj

```diff
@@ -34,12 +34,24 @@
       (throw (Exception. (format "The fields [%s] are not supported by generators with schema type [%s]"
                                  (pr-str unexpected-fields) (pr-str schema-type)))))))
 
+(defn- add-dependencies
```

Comment:
I don't fully understand the dependencies concept. This is something we're only doing in tests, not something that's in the schema? Maybe an example would help.

Reply:
Its in the schema